week1-webclient_ui.md

```diff
@@ -0,0 +1,101 @@
+# Codess in the Classroom - Week 1: The Web Client
+
+## Building the Pet Shelter UI
+
+There are several UI frameworks that can be used to have a good-looking UI with React without
+the need of manually writing style instructions.
+For instance, one can use [Bootstrap][bootstrap], [Material-UI][material-ui] or [Office UI Fabric][office-ui-fabric]. For this project, we suggest using Bootstrap, through its most popular React bindings: [react-boostrap][react-bootstrap].
+
+1. Run `npm install react-bootstrap @types/react-bootstrap` to install the `react-boostrap` library.
+2. Reference the bootstrap css file in the HTML index file located in `public/index.html` as follows.
+
+        <link
+            rel="stylesheet"
+            href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+            integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+            crossorigin="anonymous"
+        />
+
+3. Use the Bootstrap component library to build a simple view for all the animals hosted by the shelter. Here are some useful layouts and components to consider:  
+    - Grid layout system: https://react-bootstrap.github.io/layout/grid/
+    - Cards: https://react-bootstrap.github.io/components/cards/
+    - Card Layouts: https://react-bootstrap.github.io/components/cards/#card-layout
+
+
+An example of the view is shown in the image below - though, by all means, feel free to come up with anything that looks different.
```

Comment:
I wonder if something like "You can use the image below as a guide, but feel free to create your own design" might set expectations more clearly than "feel free to come up with anything different"?